common/modules/olm/manifests/1.4.0/amq-streams.v1.4.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,1214 @@
+    repository: https://github.com/strimzi/strimzi-kafka-operator
+    createdAt: 2020-02-10T10:00:00Z
+    support: Red Hat
+spec:
+  displayName: Red Hat Integration - AMQ Streams
+  description: |
+    **Red Hat AMQ Streams** is a massively scalable, distributed, and high performance data streaming platform based on the Apache Kafka® project. 
+    AMQ Streams provides an event streaming backbone that allows microservices and other application components to exchange data with extremely high throughput and low latency. 
+    
+    **The core capabilities include:**
+
+    * A pub/sub messaging model, similar to a traditional enterprise messaging system, in which application components publish and consume events to/from an ordered stream
+
+    * The long term, fault-tolerant storage of events
+
+    * The ability for a consumer to replay streams of events
+
+    * The ability to partition topics for horizontal scalability
+
+    Red Hat AMQ Streams provides a way to run an [Apache Kafka](https://kafka.apache.org) cluster on [OpenShift](https://www.openshift.com/) in various deployment configurations.
+    See the Red Hat AMQ [website](https://access.redhat.com/products/red-hat-amq) for more details about the project.
+
+    ### What is new in AMQ Streams 1.4.0
+
+    * Add support for Keycloak authorization
+
+    * Add support for Mirror Maker 2.0
+
+    * Add possibility to specify custom server certificates to TLS based listeners
+
+    * Add support for user quotas
+
+    * Expose JMX port on Kafka brokers via an internal service
+
+    * Add new KafkaConnector resource and related connector operator
+
+    * Improved Kafka rolling update
+
+    * Provide PKCS12 truststore and keystore for clients
+    
+    ### Supported Features
+
+    * **Manages the Kafka Cluster** - Deploys and manages all of the components of this complex application, including dependencies like Apache ZooKeeper® that are traditionally hard to administer.
+
+    * **Includes Kafka Connect** - Allows for configuration of common data sources and sinks to move data into and out of the Kafka cluster.
+
+    * **Topic Management** - Creates and manages Kafka Topics within the cluster.
+
+    * **User Management** - Creates and manages Kafka Users within the cluster.
+
+    * **Includes Kafka Mirror Maker** - Allows for morroring data between different Apache Kafka® clusters.
+
+    * **Includes HTTP Kafka Bridge** - Allows clients to send and receive messages through an Apache Kafka® cluster via HTTP protocol.
+
+    ### Upgrading your Clusters
+    
+    The AMQ Streams operator understands how to run and upgrade between a set of Kafka versions.
+    When specifying a new version in your config, check to make sure you are not using any features that may have been removed.
+    See [the upgrade guide](https://access.redhat.com/documentation/en-us/red_hat_amq/7.3/html/using_amq_streams_on_openshift_container_platform/index) for more information.
```

Comment:
Will this be under AMQ 7.3? Wasn't that where 1.3 is?

Reply:
good catch! Fixed! It will be 7.6 as said by doc team.